src/Worker/PushSubscription.php

```diff
@@ -82,6 +82,9 @@ public static function execute(int $sid, int $nid)
 			}
 		}
 
+		// @todo Add a good title here, see the functionality in enotify.php
```

Comment:
I still prefer @tobias 's suggestion of "meaningful" or "relevant".
```suggestion
		// @todo Add a meaningful title here, see the functionality in enotify.php
```
```suggestion
		// @todo Add a relevant title here, see the functionality in enotify.php
```